plugins/quickicon/overridecheck/overridecheck.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Quickicon.Overridecheck
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Session\Session;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Plugin\CMSPlugin;
+
+/**
+ * Joomla! update notification plugin
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class PlgQuickiconOverrideCheck extends CMSPlugin
+{
+	/**
+	 * Load the language file on instantiation.
+	 *
+	 * @var    boolean
+	 * @since  3.1
```

Comment:
__DEPLOY_VERSION__

Reply:
@laoneo @astridx Can you please explain when we use `__DEPLOY_VERSION__`.
Mean, when we create new function, class, or variable right? 
But, here I have not created this `protected $autoloadLanguage = true;` property its Joomla core functionality, I used this here. I am confused.

Reply:
@Anu1601CS 
Your are right. $autoloadLanguage is a core variable in the class CMSPlugin.
But you have created $autoloadLanguage in the class PlgQuickiconOverrideCheck. 
Please have a look in the quickicons plugin phpversioncheck. This plugin was create for Joomla version 3.7 and the variable $autoloadLanguage is since version 3.7.
The quickicons plugin joomlaupdate was create for Joomla version 3.1 and the variable $autoloadLanguage is since version 3.1.
@laoneo Please correct me if I am wrong. I put that together for myself.

Reply:
Thanks @astridx